prophyc/parsers/isar.py

```diff
@@ -72,14 +72,26 @@ def make_typedef(elem):
     elif "primitiveType" in elem.attrib and elem.get("name") not in primitive_types.values():
         return model.Typedef(elem.get("name"), primitive_types[elem.get("primitiveType")])
 
-def make_enum_member(elem):
+def make_enum_member(elem, warn):
     value = elem.get('value')
-    value = value if value != "-1" else "0xFFFFFFFF"
+    try:
+        int_value = int(value, 0)
+        if int_value < 0:
+            value = "0x{:X}".format(0x100000000 + int_value)
+    except ValueError as e:
+        if warn:
+            warn(str(e))
```

Comment:
I'm not sure if it's allowed to use identifiers as enum values in isar, but its safer if we assume its possible.
Let's assume enum like this is correct:
```
<x>
    <enum name="EEnum">
        <enum-member name="EEnum_A" value="SOME_VALUE"/>
    </enum>
</x>
```
And remove this warning:
```
prophyc: warning: invalid literal for int() with base 0: 'SOME_VALUE'
```
You can add/change a testcase to ensure that such enum value compiles correctly.